### PR TITLE
Refactoring

### PR DESCRIPTION
**Pull request by robertdolca**

### CODE REVIEW

File: app/controllers/news_controller.rb [hunk 1]
```diff
@@ -1,64 +1,41 @@
...
-  def new
-    @news = News.new
-  end
-
-  # GET /news/1/edit
-  def edit
+    render :json => @news
   end
 
   # POST /news
```
Comment (palcu) [inline app/controllers/news_controller.rb]: Remove this line.

File: app/controllers/errors_controller.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
+class ErrorsController < ApplicationController
```
Comment (hurrycane) [inline app/controllers/errors_controller.rb]: I don't think this is the right place to handle errors.

**Comment by hurrycane** [comment]
The universe would benefit if we would do more review on this kind of bigger changes.

**Comment by hurrycane** [comment]
Also you forgot to remove infoeducatie-api/vendor/assets/ - stuff

**Comment by robertdolca** [comment]
@hurrycane Agreed
What do you that would be the best approach here?

**Comment by hurrycane** [comment]
The whole vendor/assets should be removed.

File: app/controllers/errors_controller.rb (hunk 2 above)
Comment (robertdolca) [inline app/controllers/errors_controller.rb]: What do you that would be the best approach here?

**Comment by hurrycane** [comment]
@robertdolca Just some default views built using JBuilder.

**Comment by robertdolca** [comment]
That would mean to have 404.html with json content.

**Comment by hurrycane** [comment]
We can have json views too. I think that would be actually recommended rather than coupling the rendering logic with the controller one.

**Pull request merged**